platform/android/android.cmake

```diff
@@ -189,7 +176,7 @@ add_library(
     ${PROJECT_SOURCE_DIR}/platform/default/src/mbgl/text/local_glyph_rasterizer.cpp
     ${PROJECT_SOURCE_DIR}/platform/android/src/test/collator_test_stub.cpp
     ${PROJECT_SOURCE_DIR}/platform/android/src/test/number_format_test_stub.cpp
-    ${PROJECT_SOURCE_DIR}/platform/android/src/test/http_file_source_test_stub.cpp
```

Comment:
Not sure about this place - probably it is better to leave this `http_file_source_test_stub.cpp` here which basically does nothing.